docs/tutorials/speech_recognition/baidu_warp_ctc.md

```diff
@@ -0,0 +1,98 @@
+# Baidu Warp CTC with MXNet
+You can get the source code for below example [here](https://github.com/dmlc/mxnet/tree/master/example/warpctc)
+
+
+&nbsp;
+
+Baidu-warpctc is a CTC implement by Baidu which support GPU. CTC can be used with LSTM to solve lable alignment problems in many areas such as OCR, speech recognition.
+
+## Install baidu warpctc
+
+```
+  cd ~/
+  git clone https://github.com/baidu-research/warp-ctc
+  cd warp-ctc
+  mkdir build
+  cd build
+  cmake ..
+  make
+  sudo make install
+```
+
+## Enable warpctc in mxnet
+
+```
+  comment out following lines in make/config.mk
```

Comment:
Yes. In this first cut. I got all content in right place. We need to clean up/test the content now.
These contents are copy of readme files in those tutorials.